app/jobs/bulk_invoice_job.rb

```diff
@@ -7,9 +7,14 @@ class BulkInvoiceJob < ApplicationJob
 
   def perform(order_ids, filepath, options = {})
     @options = options
-    orders = sorted_orders(order_ids)
-    orders.filter!(&:invoiceable?) if OpenFoodNetwork::FeatureToggle.enabled?(:invoices,
-                                                                              current_user)
+
+    orders = Spree::Order.where(id: order_ids)
+
+    if OpenFoodNetwork::FeatureToggle.enabled?(:invoices, current_user)
+      orders = orders.invoiceable
+    end
```

Comment:
This is actually done by the reflex already and will be removed in the next commit.